Source/MacSetup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+dolphin="/Applications/Dolphin.app/Contents/MacOS/Dolphin"
+dme="./build/Dolphin-memory-engine"
```

Comment:
Is there a particular reason this is capitalized?
If its a MacOS requirement, then keep it but we are moving to standard bin name with lowercase `dolphin-memory-engine`